tests/api2/test_certs.py

```diff
@@ -264,3 +264,40 @@ def test_revoking_ca():
             assert set(c['certificate'] for c in root_ca['revoked_certs']) == check_set, root_ca
         finally:
             call('certificate.delete', cert['id'], job=True)
+
+
+def test_deleted_certs_dont_exist_on_filesystem():
+    with intermediate_certificate_authority('root_ca', 'intermediate_ca') as (root_ca, intermediate_ca):
+        with certificate_signing_request('csr_test') as csr:
+            with intermediate_certificate_authority('root_ca2', 'intermediate_ca2') as (root_ca2, intermediate_ca2):
+                # no-op
+                pass
+            with certificate_signing_request('csr_test2') as csr2:
+                pass
```

Comment:
Sorry, I still don't really understand what we are testing here. Do we create `'root_ca2'`, `'intermediate_ca2'`, `'csr_test2'`, immediately delete them and assert that corresponding files do not exist? What does the code below test then?

Reply:
Yes for the first one @themylogin. 
Regarding the code below, I am trying to just mimic the behavior where we have few certificates in both db and disk just an extra check to verify that we've them in place as required.

Reply:
Please move the code below to a separate test then. One test should test one thing (otherwise if, for example, the first part fails, we'll never know the result of the second one).